build.gradle.kts

```
@@ -133,7 +133,7 @@ subprojects {
 		api(group = "org.slf4j", name = "slf4j-api", version = "2.0.12")
 
 		// Jackson BOM
-		api(platform("com.fasterxml.jackson:jackson-bom:2.16.2"))
+		api(platform("com.fasterxml.jackson:jackson-bom:2.17.0"))
```

if we care about https://togithub.com/FasterXML/jackson-databind/issues/4428

```suggestion
		api(platform("com.fasterxml.jackson:jackson-bom:2.17.0"))

		// Jackson 2.17.0 accidentally exposes bytebuddy (fixed in 2.17.1)
		components {
			withModule("com.fasterxml.jackson.core:jackson-databind") {
				allVariants {
					withDependencies {
						removeAll { it.group == "net.bytebuddy" && it.name == "byte-buddy" }
					}
				}
			}
		}
```